samples/csharp/MicrosoftEduGraphSamples/Utilities/GlobalMethods.cs

```diff
@@ -44,9 +44,13 @@ public static void ValidateConfiguration(IConfiguration configuration)
             {
                 throw new Exception("Missing studentAccount please check appconfig.json file.");
             }
-            else if (string.IsNullOrEmpty(_config["password"]))
+            else if (string.IsNullOrEmpty(_config["teacherpassword"]))
             {
-                throw new Exception("Missing password please check appconfig.json file.");
+                throw new Exception("Missing teacher password please check appconfig.json file.");
+            }
+            else if (string.IsNullOrEmpty(_config["studentpassword"]))
```

Comment:
`studentPassword` check for all samples

Reply:
updated studentpassword to studentPassword for all instances and checked all samples